hera_mc/cm_utils.py

```diff
@@ -9,15 +9,17 @@
 from __future__ import print_function
 
 from hera_mc import mc
-import datetime
+from astropy.time import Time
+from astropy.time import TimeDelta
 import os.path
-import pytz
 
+PAST_DATE = '2000-01-01'
+FUTURE_DATE = '2025-12-31'
```

Comment:
Correct - this was just a simple way to make sure you bracket the project with dates.